Comp2023/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -135,8 +137,22 @@ private void addSmartDashboardWidgets( )
     SmartDashboard.putData("AutoDrivePathForward2", new AutoDrivePath(m_swerve, "forward2m", true));
     SmartDashboard.putData("AutoDrivePathBackward2", new AutoDrivePath(m_swerve, "backward2m", true));
 
-    // On-the-fly path generation buttons
-    SmartDashboard.putData("ResetOdometryToLimelight", new ResetOdometryToLimelight(m_swerve, m_vision, 0));
+    SmartDashboard.putData("ArmSetHeightStow", new ArmSetHeightStow(m_elbow, m_wrist));
+    SmartDashboard.putData("ArmSetHeightScoreLow", new ArmSetHeightScoreLow(m_elbow, m_wrist));
+    SmartDashboard.putData("ArmSetHeightScoreMid", new ArmSetHeightScoreMid(m_elbow, m_wrist));
+    SmartDashboard.putData("ArmSetHeightScoreHigh", new ArmSetHeightScoreHigh(m_elbow, m_wrist));
+    SmartDashboard.putData("AutoPreloadAndLeaveCommunity",
```

Comment:
All of our auto commands that reference other subsystems should use the same order as parameters. Omitting those that aren't needed (if any). The order so far has been m_swerve, m_elbow, m_extension, m_wrist, and m_gripper. We should keep the same pattern in these commands.